modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/application/publisher/APIApplicationPublisherImpl.java

```diff
@@ -186,6 +193,9 @@ private void _unregisterServiceRegistrations(
 	@Reference
 	private EndpointHelper _endpointHelper;
 
+	private final Map<String, EndpointMatcher> _endpointMatchersMap =
```

Comment:
I think `_endpointMatchers` whould be more appropriate based on BC's renamings

Reply:
Mmmm that is true. Well, leave it as it is then and see what he thinks this time :crossed_fingers: :trollface: